packages/components/src/components/form/Select/index.tsx

```diff
@@ -51,15 +52,15 @@ const selectStyle = (
             height,
             borderRadius: `${borderRadius}px`,
             borderWidth: `${borderWidth}px`,
-            borderColor: theme.STROKE_GREY,
+            borderColor: selectState ? getStateColor(selectState, theme) : theme.STROKE_GREY,
```

Comment:
nitpick: would be nice to define `const borderColor = selectState ? getStateColor(selectState, theme) : theme.STROKE_GREY` above and use it here and for hover/focus state

Reply:
Done.